lib/datetime_picker_formfield.dart

```diff
@@ -288,7 +309,7 @@ class _DateTimeFieldState extends FormFieldState<DateTime> {
   }
 
   void _hideKeyboard() {
-    Future.microtask(() => FocusScope.of(context).requestFocus(FocusNode()));
+    Future.microtask(() => FocusScope.of(context).unfocus());
```

Comment:
This doesn't actually work to unfocus the field. It is called when the widget gains focus and that seems to be why.

Reply:
Can you give me a reference to the issue?  I would like to read up on it since the strategy being used to dismiss the controller was replaced with the unfocus method.  I haven’t had any issues using it either.

Reply:
I've not seen it elsewhere but when I use `unfocus()` here the widget breaks.

Reply:
I have been using it here and not having an issue but I am using the latest flutter releases.  I was an issue on previous releases.  Can you give some example code it happens on for you and I will see if happens for me as well.

Reply:
I removed this from the PR.